engine/installation/linux/oracle.md

```diff
@@ -1,204 +1,236 @@
 
-To create the `docker` group and add your user:
+    ```bash
+    $ sudo yum -y install docker-engine
+    ```
 
-1. Log into Oracle Linux as a user with `sudo` privileges.
+    > **Warning**: If you have both stable and unstable repositories enabled,
+    > installing or updating Docker without specifying a version in the
+    > `yum install` or `yum upgrade` command will always install the highest
+    > available version, which will almost certainly be an unstable one.
 
-2. Create the `docker` group.
+3.  On production systems, you should install a specific version of Docker
+    instead of always using the latest. List the available versions.
+    This example uses the `sort -r` command to sort the results by version
+    number, highest to lowest.
 
-        $ sudo groupadd docker
+    > **Note**: This `yum list` command only shows binary packages. To show
+    > source packages as well, omit the `.x86_64` from the package name.
 
-3. Add your user to `docker` group.
+    ```bash
+    $ yum list docker-engine.x86_64  --showduplicates |sort -r
 
-        $ sudo usermod -aG docker username
+    docker-engine.x86_64  1.13.0-1.el7                               docker-main
+    docker-engine.x86_64  1.13.0-0.5.rc5.el7                         docker-testing
+    docker-engine.x86_64  1.13.0-0.4.rc4.el7                         docker-testing
+    docker-engine.x86_64  1.13.0-0.3.rc3.el7                         docker-testing
+    docker-engine.x86_64  1.13.0-0.2.rc2.el7                         docker-testing
+    docker-engine.x86_64  1.13.0-0.1.rc1.el7                         docker-testing
+    docker-engine.x86_64  1.12.5-1.el7                               docker-main   
+    docker-engine.x86_64  1.12.4-1.el7                               docker-main   
+    docker-engine.x86_64  1.12.3-1.el7                               docker-main   
+    docker-engine.x86_64  1.12.2-1.el7                               docker-main   
+    docker-engine.x86_64  1.12.1-1.el7                               docker-main   
+    docker-engine.x86_64  1.12.0-1.el7                               docker-main   
+    ```
 
-4. Log out and log back in.
+    The contents of the list depend upon which repositories you have enabled,
+    and will be specific to your version of Oracle Linux (indicated by the
+    `.el7` suffix on the version, in this example). Choose a specific version to
+    install. The second column is the version string. The third column is the
+    repository name, which indicates which repository the package is from and by
+    extension extension its stability level. To install a specific version,
+    append the version string to the package name and separate them by a hyphen
+    (`-`):
 
-    This ensures your user is running with the correct permissions.
+    ```bash
+    $ sudo yum -y install docker-engine-<VERSION_STRING>
+    ```
 
-5. Verify your work by running `docker` without `sudo`.
+    The Docker daemon starts automatically.
```

Comment:
This may have changed in Docker 1.13, but up to 1.12, the daemon is not started automatically in the standard RPM. You need to use `systemctl` to both enable and start the Docker daemon (on OL7) or the `service` and `chkconfig` commands to start the daemon on OL6.